src/databricks/sqlalchemy/__init__.py

```diff
@@ -31,6 +33,141 @@ class DatabricksImpl(DefaultImpl):
         __dialect__ = "databricks"
 
 
+import logging
+
+logger = logging.getLogger(__name__)
+
+DBR_LTE_12_NOT_FOUND_STRING = "Table or view not found"
+DBR_GT_12_NOT_FOUND_STRING = "TABLE_OR_VIEW_NOT_FOUND"
+
+
```

Comment:
Why is all of this stuff in __init__.py?

Reply:
It's a biproduct of earlier decisions. I'll move it out of __init__.py in a follow-up PR.

Reply:
The guidance from sqlalchemy is that this should be in a file called `base.py`.